app/views/admin/alert_types/show.html.erb

```diff
@@ -8,14 +8,19 @@
   <dd class="col-md-10"><%= @alert_type.frequency.humanize %></dd>
 
   <dt class="col-md-2">Show ratings</dt>
-  <dd class="col-md-10"><%= @alert_type.show_ratings %></dd>
```

Comment:
I don't think we use `show_ratings` for anything. Can we remove it too?

Reply:
Yes, we can